system/Database/SQLite3/Result.php

```diff
@@ -95,7 +95,8 @@ public function getFieldData(): array
 		];
 
 		$retVal = [];
-
+        $this->resultID->fetchArray(SQLITE3_NUM);  
```

Comment:
Please align the indent with line 97.

Reply:
I figured out why, it had spaces infront that why I keep on beying confused as it looks correct for me and then it diff.
Before

![prent](https://user-images.githubusercontent.com/33496470/88203141-26dec000-cc4a-11ea-9909-9ac4b5c1f550.png)

After
![prent](https://user-images.githubusercontent.com/33496470/88203296-365e0900-cc4a-11ea-9d49-1250029a9386.png)